iFixit/formy
# Update Formy to React v16.4.1

## Details
In order for us to upgrade our code base to React v16.4.1, we first have to upgrade its dependencies to versions that support React 16. This pull upgrades Formy to React v16 and certain dependencies to versions that support the latest version of React.

## QA
- Checkout this branch on your local machine
- Look through all our components and compare them to the master branch
- Ensure that there are no issues
- Done!
